library_models.js

```diff
@@ -0,0 +1,40 @@
+/*
+Exercise 5
+A JavaScript program that models a library, and should have a function that will add, remove and also list the proparties in it.
```

Comment:
Hello @Dove-white, please read the question carefully again to understand what you are suppose to do. 

- [ ] Add a book to the library: The program should allow the user to add books to the library. Each book should have a title, author, and year of publication. :heavy_check_mark: 
- [ ] Remove a book from the library: The program should allow the user to remove a book from the library based on its title. :heavy_check_mark: 
- [ ] Get the count of books in the library: The program should provide the total count of books in the library. :negative_squared_cross_mark: 
- [ ] List all the books in the library: The program should list all the books in the library with their details (title, author, and year of publication). :negative_squared_cross_mark:

Reply:
Thanks for the corrections 👍